Dockerfile

```diff
@@ -2,6 +2,11 @@ FROM sbtscala/scala-sbt:eclipse-temurin-17.0.4_1.7.2_2.13.10
 
 WORKDIR /opt/test-runner
 
+ENV CONFIG_PATH '.meta'
```

Comment:
Why is this configurable via an ENV variable?

Reply:
As in "Why configurable"? Or "Why via ENV and not ARG"? Or something else?

I made it configurable to easily test the run script with exercises checked out via exercism CLI

Reply:
> I made it configurable to easily test the run script with exercises checked out via exercism CLI

That was indeed why I meant :)